eth_account/_utils/transaction_utils.py

```diff
@@ -51,7 +61,7 @@ def _access_list_rpc_to_rlp_structure(access_list: Sequence) -> Sequence:
                 tuple(_ for _ in d['storageKeys'])  # tuple of storage key values
```

Comment:
I can't leave a comment on line 50, but we may need to `cast(ChecksumAddress, d['address'])` there to get the other types working. If you can figure out how to do it without the cast, that would be even better!